### PR TITLE
Release v0.5.0.post2

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: v0.5.0.post2

## Release details

Packages:
- Fixes synchronized data for safe deployment
- Ensures that the synchronized data class is set everywhere and added a warning when it is not

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] I've updated the dependencies versions to the latest, wherever is possible.
- [ ] Lint and unit tests pass locally (please run tests also manually, not only with `tox`)
- [ ] I built the documentation and updated it with the latest changes
- [ ] I've added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `__init__.py` file.
- [ ] I published the latest version on TestPyPI and checked that the following command work:
       ```pip install project-name==<version-number> --index-url https://test.pypi.org/simple --force --no-cache-dir --no-deps```
- [ ] After merging the PR, I'll publish the build also on PyPI. Then, I'll make sure the following
      command will work:
      ```pip install project-name-template==<version_number> --force --no-cache-dir --no-deps```  
- [ ] After merging the PR, I'll tag the repo with `v${VERSION_NUMVER}` (e.g. `v0.1.2`)

### CODE REVIEW

**Review by Adamantios — APPROVED** [review]

File: HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
 
 
+# 0.5.0.post2 (2022-12-09)
+
+Packages:
+- Fixes synchronized data for safe deployment
+- Ensures that the synchronized data class is set everywhere and added a warning when it is not
+
+
 # 0.5.0.post1 (2022-12-08)
```
Comment (DavidMinarsch) [inline HISTORY.md:11]: Missing here: https://github.com/valory-xyz/open-autonomy/pull/1629

Comment (DavidMinarsch) [reply HISTORY.md:11]: Also: https://github.com/valory-xyz/open-autonomy/pull/1628

Comment (DavidMinarsch) [reply HISTORY.md:11]: Also: https://github.com/valory-xyz/open-autonomy/pull/1621

File: HISTORY.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 # Release History - `open-autonomy`
 
 
+# 0.5.0.post2 (2022-12-09)
+
+Packages:
```
Comment (DavidMinarsch) [inline HISTORY.md:6]: Missing:
- https://github.com/valory-xyz/open-autonomy/pull/1635
- https://github.com/valory-xyz/open-autonomy/pull/1633

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
It's important all PRs are considered between post releases. Otherwise we repeat the problem from open-aea!

**Review by DavidMinarsch — APPROVED** [review]

**Review by Karrenbelt — APPROVED** [review]

**Pull request merged**